integration/out-of-bounds-read.test.ts

```diff
@@ -0,0 +1,81 @@
+import { describe, before, test } from 'mocha'
+import { expect } from 'chai'
+import express from 'express'
+import { CreateHttpServer } from '../src/server'
+import {
+  getOutOfBoundsReadCheri,
+  getOutOfBoundsReadAarch64,
+  getInvalidExecutable,
+} from '../integration/helper/routeHelper'
+
+describe('Tests aarch64 version', () => {
+  let app: express.Express
+
+  before(async function () {
+    app = await CreateHttpServer()
+  })
+
+  test('Happy Path - aarch64', async () => {
+    const response = await getOutOfBoundsReadAarch64(app, 'pass', -32, -28)
+
```

Comment:
maybe assert `response.body.output` is revealing the password `Storing Secret...\nThe data between indices -32 and -28 is: pass\n'`